.changeset/3b448832/changes.md

```diff
@@ -0,0 +1 @@
+- Added support for module resolutions in the extract react types loader
```

Comment:
This reads a bit misleading. The change is 'support loading modules from `src` as well as `node_modules`.